libs/keystore/key_access_win.go

```diff
@@ -0,0 +1,30 @@
+//go:build windows
+
+package keystore
+
+import (
+	"fmt"
+	"golang.org/x/sys/windows"
+	"unsafe"
+)
+
+// keyAccess checks whether file is not accessible by all users.
+func keyAccess(path string) error {
+
+	// Check the file's DACL
+	securityDescriptor, err := windows.GetNamedSecurityInfo(path, windows.SE_FILE_OBJECT, windows.DACL_SECURITY_INFORMATION)
+	if err != nil {
+		return fmt.Errorf("Error getting file's DACL: %v\n", err)
+	}
+	defer func() {
+		if err != nil {
+			_, _ = windows.LocalFree((windows.Handle)(unsafe.Pointer(securityDescriptor)))
+		}
+	}()
+	_, _, err = securityDescriptor.DACL()
+	if err != nil {
+		return fmt.Errorf("File is too Open with no DACL: %v\n", err)
```

Comment:
yes, this is a problem, but I believe it's not serious at the beginning, and we could continue to improve it later. I explained in the issue tracker of this PR. because the go package of win32 api has some issues currently due to the special structure of Win, I have to import a lot of win32 api by myself to achieve better constraints, which could be even more risky than current. Because Win isn't a multi-user environment, it usually doesn't need such a check as on linux in convention.